.gitignore

```diff
@@ -14,6 +14,9 @@
 #################################
 /nbproject
 .idea
+/.project
+/.buildpath
+/.settings/
```

Comment:
As the comment says "# IDE and editor specific files #" this should be the correct place for them.